mwaa-local-env

```diff
@@ -76,6 +79,15 @@ reset-db)
 start)
    docker-compose -f ./docker/docker-compose-local.yml up
    ;;
+start-daemon)
+   docker-compose -f ./docker/docker-compose-local.yml up -d
+   ;;
+down)
+   docker-compose -f ./docker/docker-compose-local.yml down
+   ;;
+stop)
+   docker-compose -f ./docker/docker-compose-local.yml stop
```

Comment:
I am not convinced to have ambiguity with the two commands. The reason why this CLI exists is to mask the docker compose stuff from regular user. Down and stop assumed the user knows docker compose.

I would prefer to just have one command stop, and use the more cleanup option of the two. rest of PR is fine to me

Reply:
I see the above two updates are convenience when using this repo to do development. Even if you are a basic user of docker or know your way around. The above to me is what I use the most as a developer. I believe to develop in docket or using this repo you must have 
Some backgrounds and understanding of how docker works. Even basics understanding which I believe is what stop and down command is used in docker. As a developer it's not practical for me to start and stop each time. As you sometimes have to switch gears and use the down command to work on other tasks to free up your memory and be able to comeback and continue where you left off having all settings and logs present without having to start all over again with a start docker airflow each time.

Reply:
After some review. I realize why we may not want to add "down" command because we have this [syntax](https://github.com/aws/aws-mwaa-local-runner/blob/a0bf95cce76d0e8dd17a57d892a26de767d3dde6/docker/docker-compose-local.yml#L14) in docker-compose-local.yml which save the metadata of the instance locally. I believe we should remove this because you cannot actually "reset" your instance when you do docker stop. For Example, a user do development and wanted to change their test_dag.py dag name but because the docker instance above always save the metadata locally. Even if the user thinks that he reset the docker instance when he start back docker he'll encounter the same issue where his dag name change will not be change due to this metadata is being loaded back into the new docker instance. I didnt realize this as well until I saw that my metadata is being saved locally and in order to have my instance reset (which what I always do when developing and want to start a fresh and remove cache or bug due to cache) I have to remove the above line from my docker-compose-local.yml. 




the primarily because when we stop (shutdown docker) the assumption was that everything should be reset when we start the docker back up again. but, this is not the case for this docker docker-compose-local.yml